documentation/modules/auxiliary/dos/http/ws_dos.md

```diff
@@ -0,0 +1,61 @@
+## Vulnerable Application
+ 
+   [ws < 1.1.5 || (2.0.0 , 3.3.1)]
+   (https://nodesecurity.io/advisories/550)
```

Comment:
The newline and spaces are breaking the markdown.

![image](https://user-images.githubusercontent.com/434827/33816654-8ef00a2a-de8e-11e7-805b-add2c52460ee.png)